### PR TITLE
Refs[TASK][39468] Tạo giao diện cho Menu

### PR DESCRIPTION
**Pull request by okuloj**
## Related Tickets
- [#Task 39468: Tạo giao diện cho Menu](https://edu-redmine.sun-asterisk.vn/issues/39468#change-165213)
## WHAT
- Tạo giao diện cho Menu
## Evidence (Screenshot or Video)
Screenshot_1628492763

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.edu.news.MainActivity">
+    tools:context="com.edu.news.screen.MainActivity">
 
-</android.support.constraint.ConstraintLayout>
+    <com.google.android.material.bottomnavigation.BottomNavigationView
+        android:id="@+id/bottomNavigationView"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
```
Comment (luongth-2520) [inline app/src/main/res/layout/activity_main.xml]: đưa dp vào dimens

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+        app:itemBackground="@android:color/white"
+        app:itemIconTint="@android:color/black"
+        app:itemTextColor="@android:color/black"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:menu="@menu/bottom_nav_menu"/>
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (luongth-2520) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
        app:menu="@menu/bottom_nav_menu"/>
</androidx.constraintlayout.widget.ConstraintLayout>
```
remove space

File: app/src/main/res/menu/bottom_nav_menu.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        android:id="@+id/itemFavorite"
+        android:title="@string/favorite"
+        android:icon="@drawable/ic_baseline_star_24"/>
+
+    <item
+        android:id="@+id/itemFollows"
+        android:title="@string/follows"
+        android:icon="@drawable/ic_baseline_favorite_24"/>
+
+</menu>
```
Comment (luongth-2520) [inline app/src/main/res/menu/bottom_nav_menu.xml]:
```suggestion
        android:icon="@drawable/ic_baseline_favorite_24"/>
</menu>
```

File: app/src/main/res/menu/bottom_nav_menu.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/itemHome"
```
Comment (luongth-2520) [inline app/src/main/res/menu/bottom_nav_menu.xml:5]:
```suggestion
    <item
        android:id="@+id/home"
```

File: app/src/main/res/menu/bottom_nav_menu.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        android:title="@string/home"
+        android:icon="@drawable/ic_baseline_home_24"/>
+
+    <item
+        android:id="@+id/itemFavorite"
```
Comment (luongth-2520) [inline app/src/main/res/menu/bottom_nav_menu.xml:10]:
```suggestion
    <item
        android:id="@+id/favorite"
```

File: app/src/main/res/menu/bottom_nav_menu.xml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        android:title="@string/favorite"
+        android:icon="@drawable/ic_baseline_star_24"/>
+
+    <item
+        android:id="@+id/itemFollows"
```
Comment (luongth-2520) [inline app/src/main/res/menu/bottom_nav_menu.xml:15]:
```suggestion
    <item
        android:id="@+id/follows"
```

**Review by luongth-2520 — APPROVED** [review]

File: app/src/main/res/menu/bottom_nav_menu.xml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/home"
```
Comment (tuanlvt-1117) [inline app/src/main/res/menu/bottom_nav_menu.xml]: tương tự các item còn lại
```suggestion
        android:id="@+id/itemHome"
```

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**